README.md

```diff
@@ -96,6 +96,14 @@ nodejs - nodejs-myproject.192.168.42.147.nip.io
 $ curl nodejs-myproject.192.168.42.147.nip.io
 ```
 
+## Community, Discussion, Contribution and Support
+
+**Chat:** We have a public channel [#Odo on chat.openshift.io](https://chat.openshift.io/).
```

Comment:
can we use a direct link to the channel? - https://chat.openshift.io/developers/channels/odo